Update controllerClient information (#13639)
nvaccess/nvda: extras/controllerClient/readme.md

  - A C header file containing declarations for all the provided functions.

The **`extras/controllerClient/x86` directory** also contains example usage:

* `example_python.py`
  - an example Python program that uses the NVDA controller client API.
* `example_c.c`
  - The source code for an example C program that uses the NVDA controller client API.
* `example_csharp.cs`
  - The source code for an example C# program that uses the NVDA controller client API.

## Available Functions

All functions in this dll return 0 on success and a non-0 code on failure. Failure could be due to not being able to communicate with NVDA or incorrect usage of the function. The error codes that these functions return are always standard Windows error codes.

For definitions of all the functions available, please see `nvdaController.h`.
The functions are documented with comments.

## License

The NVDA Controller Client API is licensed under the GNU Lesser General Public License (LGPL), version 2.1.
In simple terms, this means you can use this library in any application, but if you modify the library in any way, you must contribute the changes back to the community under the same license.

> Please see `license.txt` in this directory for more details.

Reviewer: ```suggestion
Please refer to [license.txt](./license.txt) for more details.
```

## What is Included?

**Note:**

The header and library files are built along with nvdaHelper.
If you want to build them yourself, please see `readme.txt` in the NVDA source distribution for instructions.

> Versions of the libraries and headers are supplied for both `x86` and `x64` applications. They have been split in to `x86` and `x64` directories, respectivly.

Reviewer 2: respectivly -> respectively
```suggestion
Versions of the libraries and headers are supplied for both `x86` and `x64` applications. They have been split in to `x86` and `x64` directories, respectively.
```

> Versions of the libraries and headers are supplied for both `x86` and `x64` applications. They have been split in to `x86` and `x64` directories, respectivly.

Reviewer 2: For some time now we also compile Controller Client for ARM64. Shouldn't it be mentioned as a supported architecture here?
